### PR TITLE
More focus on Plone 6 in README

### PR DESCRIPTION
**Pull request by jensens**
In hindsight of a final release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,16 +19,11 @@
 Plone backend [Docker](https://docker.com) images using Python 3 and [pip](https://pip.pypa.io/en/stable/).
 
 > **NOTE**:
```
Comment (stevepiercy) [inline README.md]: `>` is a block quote or a quotation of a paragraph. I would remove all of these from the README. Also the colon should have the same formatting as the word "Note".

```suggestion
**Note:**
```

File: README.md [hunk 2]
```diff
@@ -19,16 +19,11 @@
...
-> These images are not officially supported by the Plone Community for Plone 5.
-> They will become the official ones for the upcoming Plone 6 release (together with [plone.frontend](https://github.com/plone/plone-frontend)).
-> These image are **not** buildout based!
+> These images are the official ones for the Plone 6 release (together with [plone.frontend](https://github.com/plone/plone-frontend)).
```
Comment (stevepiercy) [inline README.md]:
```suggestion
These are the official images for the Plone 6 release, together with [plone.frontend](https://github.com/plone/plone-frontend).
```

File: README.md [hunk 3]
```diff
@@ -19,16 +19,11 @@
...
+> These image are **not** Buildout based!
```
Comment (stevepiercy) [inline README.md]: This begs the question, what are they based on? Or do you mean "not compatible with buildout"?

```suggestion
These images are **not** Buildout based!
```

File: README.md [hunk 4]
```diff
@@ -19,16 +19,11 @@
...
 
 ## Supported tags and respective Dockerfile links
 
-### Plone 5.2 (Stable Plone)
-
-- `latest, 5, 5.2, 5.2.9` [(5.2.x/Dockerfile)](https://github.com/plone/plone-backend/blob/v5.2.9/Dockerfile)
-
 ### Plone 6 (Next Plone Release)
 
 - `6.0.0rc1` [(6.0.x/Dockerfile)](https://github.com/plone/plone-backend/blob/v6.0.0rc1/Dockerfile)
```
Comment (stevepiercy) [inline README.md]: Why use a list for a single item, when a paragraph does the job?

```suggestion
`6.0.0rc1` [(6.0.x/Dockerfile)](https://github.com/plone/plone-backend/blob/v6.0.0rc1/Dockerfile)
```

File: README.md [hunk 5]
```diff
@@ -37,6 +32,14 @@ Plone backend [Docker](https://docker.com) images using Python 3 and [pip](https
 
 - `nightly` [(Dockerfile.nightly)](https://github.com/plone/plone-backend/blob/v5.2.7/Dockerfile.nightly)
```
Comment (stevepiercy) [inline README.md]:
```suggestion
`nightly` [(Dockerfile.nightly)](https://github.com/plone/plone-backend/blob/v5.2.7/Dockerfile.nightly)
```

File: README.md [hunk 6]
```diff
@@ -37,6 +32,14 @@ Plone backend [Docker](https://docker.com) images using Python 3 and [pip](https
...
 
+### Plone 5.2 (Previous Stable Plone)
+
+> **NOTE**:
```
Comment (stevepiercy) [inline README.md]:
```suggestion
**Note:**
```

File: README.md [hunk 7]
```diff
@@ -37,6 +32,14 @@ Plone backend [Docker](https://docker.com) images using Python 3 and [pip](https
...
+> These images are **not** officially supported by the Plone Community for Plone 5.
```
Comment (stevepiercy) [inline README.md]:
```suggestion
These images for Plone 5 are **not** officially supported by the Plone community.
```

File: README.md [hunk 8]
```diff
@@ -37,6 +32,14 @@ Plone backend [Docker](https://docker.com) images using Python 3 and [pip](https
...
+
+- `latest, 5, 5.2, 5.2.9` [(5.2.x/Dockerfile)](https://github.com/plone/plone-backend/blob/v5.2.9/Dockerfile)
```
Comment (stevepiercy) [inline README.md]:
```suggestion
`latest, 5, 5.2, 5.2.9` [(5.2.x/Dockerfile)](https://github.com/plone/plone-backend/blob/v5.2.9/Dockerfile)
```

File: README.md [hunk 9]
```diff
@@ -37,6 +32,14 @@ Plone backend [Docker](https://docker.com) images using Python 3 and [pip](https
...
+
+There are official [Buildout based Plone 5 images](https://hub.docker.com/_/plone) available too.
```
Comment (stevepiercy) [inline README.md]:
```suggestion
See also official [Buildout based Plone 5 images](https://hub.docker.com/_/plone).
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
There is some unconventional and non-semantic Markdown being used. I also fixed up some English grammar.

**Review by ericof — APPROVED** [review]
LGTM, thanks for taking care of that

**Review by Rudd-O — APPROVED** [review]

File: README.md [hunk 10]
```diff
@@ -18,25 +18,28 @@
...
 
-- `nightly` [(Dockerfile.nightly)](https://github.com/plone/plone-backend/blob/v5.2.7/Dockerfile.nightly)
+`nightly` [(Dockerfile.nightly)](https://github.com/plone/plone-backend/blob/v5.2.7/Dockerfile.nightly)
+
+### Plone 5.2 (Previous Stable Plone)
+
+**Note:**
+These images for Plone 5 are **not** officially supported by the Plone community.
+
+`latest, 5, 5.2, 5.2.9` [(5.2.x/Dockerfile)](https://github.com/plone/plone-backend/blob/v5.2.9/Dockerfile)
```
Comment (stevepiercy) [inline README.md]: These are separate values, not one big one.

Also I want to verify that `latest` still points to v5 at this time. The terms used in the foregoing imply Plone 6 is released and "latest", and we say "Plone 5.2 (Previous Stable Plone)". I understand if we are in flux at the moment, but we should reflect the absolute current state now, not the future or previous states.

```suggestion
`latest`, `5`, `5.2`, `5.2.9` [(5.2.x/Dockerfile)](https://github.com/plone/plone-backend/blob/v5.2.9/Dockerfile)
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
One suggestion, and a question for clarification, please. Thank you!

**Comment by stevepiercy** [comment]
This PR needs a changelog entry, please!

File: README.md [hunk 11]
```diff
@@ -18,25 +18,28 @@
 
 Plone backend [Docker](https://docker.com) images using Python 3 and [pip](https://pip.pypa.io/en/stable/).
 
-> **NOTE**:
-> These images are not officially supported by the Plone Community for Plone 5.
-> They will become the official ones for the upcoming Plone 6 release (together with [plone.frontend](https://github.com/plone/plone-frontend)).
-> These image are **not** buildout based!
+**Note:**
+These are the official images for the Plone 6 release, together with [plone.frontend](https://github.com/plone/plone-frontend).
```
Comment (gforcada) [inline README.md]: dot or dash? `plone.frontend` vs `plone-frontend` 😄

Comment (ericof) [inline README.md]:
```suggestion
These are the official images for the Plone 6 release, together with [plone-frontend](https://github.com/plone/plone-frontend).
```

Comment (stevepiercy) [reply README.md]: This change was not committed. Unresolving.

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Comment by ericof** [comment]
> This PR needs a changelog entry, please!

We do not have a changelog on this repo, and I'm open to discussing how/when we implement one

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
> We do not have a changelog on this repo, and I'm open to discussing how/when we implement one

Oh, mr.roboto complains, which is why I mentioned it. If a change log is not needed, then that should be silenced.

**Comment by gforcada** [comment]
@stevepiercy then we want to merge and deploy this: https://github.com/plone/mr.roboto/pull/98 😄

**Pull request merged**